include/swift/ABI/MetadataValues.h

```diff
@@ -929,12 +929,17 @@ class FieldType {
 enum class ExclusivityFlags : uintptr_t {
   Read             = 0x0,
   Modify           = 0x1,
-  // Leave space for other actions.
-  // Don't rely on ActionMask in stable ABI.
+  // ActionMask can grow without breaking the ABI because the runtime controls
+  // how these flags are encoded in the "value buffer". However, any additional
+  // actions must be compatible with the original behavior for the old, smaller
+  // ActionMask (older runtimes will continue to treat them as either a simple
+  // Read or Modify).
   ActionMask       = 0x1,
 
   // Downgrade exclusivity failures to a warning.
-  WarningOnly      = 0x10
+  WarningOnly      = 0x10,
+  // The runtime should track this access to check against subsequent accesses.
+  Tracking         = 0x20
```

Comment:
I talked myself into agreeing with you that this is the right polarity for this flag.  You should document how this flag changes the expectation about endAccess, though.